diff --git a/dist/server/lib/router-server.js b/dist/server/lib/router-server.js
index fec3014838b1c9337408fa53c20f5020ae89bb10..23495acb51aa3ea9762c14b85621067ffffeea92 100644
--- a/dist/server/lib/router-server.js
+++ b/dist/server/lib/router-server.js
@@ -459,9 +459,9 @@ async function initialize(opts) {
             });
             // TODO: allow upgrade requests to pages/app paths?
             // this was not previously supported
-            if (matchedOutput) {
-                return socket.end();
-            }
+            // if (matchedOutput) {
+            //     return socket.end();
+            // }
             if (parsedUrl.protocol) {
                 return await (0, _proxyrequest.proxyRequest)(req, socket, parsedUrl, head);
             }
